02.toolchain/commonbuild-doxygen.sh: No such file or directory

Something went wrong after #23 merge :(
Example 02.toolchain-basic doesn't work as expected (02.toolchain-complex works perfectly).

```
2020-07-09 21:32:20 [INFO]: Task "unpack-python-src" started.
2020-07-09 21:32:22 [INFO]: Task "unpack-python-src" terminated.

2020-07-09 21:32:22 [INFO]: Task "check-all-src-dirs" started.
Checking if "/tmp/makbet/examples/02.toolchain-basic/doxygen/doxygen-Release_1_8_18/" directory exists...
/tmp/makbet/examples/02.toolchain-basic/doxygen/doxygen-Release_1_8_18/ -> Directory found.
Checking if "/tmp/makbet/examples/02.toolchain-basic/git/git-2.27.0/" directory exists...
/tmp/makbet/examples/02.toolchain-basic/git/git-2.27.0/ -> Directory found.
Checking if "/tmp/makbet/examples/02.toolchain-basic/kcov/kcov-38/" directory exists...
/tmp/makbet/examples/02.toolchain-basic/kcov/kcov-38/ -> Directory found.
Checking if "/tmp/makbet/examples/02.toolchain-basic/make/make-4.3/" directory exists...
/tmp/makbet/examples/02.toolchain-basic/make/make-4.3/ -> Directory found.
Checking if "/tmp/makbet/examples/02.toolchain-basic/python/Python-3.9.0b4/" directory exists...
/tmp/makbet/examples/02.toolchain-basic/python/Python-3.9.0b4/ -> Directory found.
2020-07-09 21:32:22 [INFO]: Task "check-all-src-dirs" terminated.

2020-07-09 21:32:22 [INFO]: Task "build-doxygen" started.
make: /home/user/makbet/examples/lib/tasks/02.toolchain/commonbuild-doxygen.sh: No such file or directory
make: *** [Makefile:158: /home/user/makbet/.makbet/events/cfg/build-doxygen.terminated.cfg] Error 127
```

--- a/examples/02.toolchain-basic/Makefile
+++ b/examples/02.toolchain-basic/Makefile
@@ -140,35 +140,35 @@ $(eval $(call TASK_template, \
 $(eval $(call TASK_template, \
 	build-doxygen, "Build doxygen inside its source dir", \
 	check-all-src-dirs, \
-	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/commonbuild-doxygen.sh, \
+	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/common/build-doxygen.sh, \
 		$(WORK_DIR)/doxygen/doxygen-Release_1_8_18/ \
 ))
 
 $(eval $(call TASK_template, \
 	build-git, "Build git inside its source dir", \
 	build-doxygen, \
-	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/commonbuild-git.sh, \
+	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/common/build-git.sh, \
 		$(WORK_DIR)/git/git-2.27.0/ \
 ))
 
 $(eval $(call TASK_template, \
 	build-kcov, "Build kcov inside its source dir", \
 	build-git, \
-	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/commonbuild-kcov.sh, \
+	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/common/build-kcov.sh, \
 		$(WORK_DIR)/kcov/kcov-38/ \
 ))
 
 $(eval $(call TASK_template, \
 	build-make, "Build GNU make inside its source dir", \
 	build-kcov, \
-	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/commonbuild-make.sh, \
+	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/common/build-make.sh, \
 		$(WORK_DIR)/make/make-4.3/ \
 ))
 
 $(eval $(call TASK_template, \
 	build-python, "Build python inside its source dir", \
 	build-make, \
-	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/commonbuild-python.sh, \
+	$(MAKBET_PATH)/examples/lib/tasks/02.toolchain/common/build-python.sh, \
 		$(WORK_DIR)/python/Python-3.9.0b4/ \
 ))
 

